### PR TITLE
chore (kornia-py): makefile w/ uv, and extend python ci

### PR DESCRIPTION
**Pull request by johnnv1**

### CODE REVIEW

File: kornia-py/Makefile
Comment (edgarriba) [inline kornia-py/Makefile:1]: @johnnv1 I was actually planning to replace Makefile by `justfile` as I've seen recently some russt projects adopting it

Comment (edgarriba) [reply kornia-py/Makefile:1]: https://github.com/casey/just

Comment (johnnv1) [reply kornia-py/Makefile:1]: kinda nice, i migrated this small one related to the py env to test it first... later one i can check the main makefile

**Review by edgarriba — APPROVED** [review]

**Comment by edgarriba** [comment]
@johnnv1 if you are up to -- you can give a shot to replace also the top level Makefile && update readme ?

**Pull request merged**